src/multiple_choice/template.py

```diff
@@ -511,16 +511,11 @@ def addModel(col):
 
 def updateTemplate(col):
     """Update add-on card templates"""
-    print("Updating %s card template".format(kprim_model))
-    model = col.models.byName(kprim_model)
+    print("Updating %s card template".format(aio_model))
+    model = col.models.byName(aio_model)
     template = model['tmpls'][0]
     template['qfmt'] = card_front
     template['afmt'] = card_back
     model['css'] = card_css
     col.models.save()
```

Comment:
In my tests I needed to add `models` to `col.models.save()` like I did in [my PR](https://github.com/zjosua/anki-mc/pull/42/files).

Also my python formatter told me that something's wrong/outdated with the `"...".format()` so I recommend to use the pretty formatting string as well.

Reply:
Thanks for pointing this out.